tests/test_MultilabelBinaryPolicy.py

```diff
@@ -0,0 +1,794 @@
+    ]
+    result = policy.negative_examples("2.1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_less_inclusive_policy_negative_examples_1(digraph, features_1d, labels):
+    policy = LessInclusivePolicy(digraph, features_1d, labels)
+    ground_truth = [
+        False,
+        False,
+        True,
+        True,
+        True,
+        True,
+        True,
+        True,
+        # Mutli-label Test cases
+        False,
+        False,
+        True,
+    ]
+    result = policy.negative_examples("1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_less_inclusive_policy_negative_examples_2(digraph, features_1d, labels):
+    policy = LessInclusivePolicy(digraph, features_1d, labels)
+    ground_truth = [
+        True,
+        True,
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        # Mutli-label Test cases
+        True,
+        False,
+        False,
+    ]
+    result = policy.negative_examples("2")
+    assert_array_equal(ground_truth, result)
+
+
+def test_less_inclusive_policy_negative_examples_3(digraph, features_1d, labels):
+    policy = LessInclusivePolicy(digraph, features_1d, labels)
+    ground_truth = [
+        True,
+        True,
+        True,
+        True,
+        False,
+        True,
+        True,
+        True,
+        # Mutli-label Test cases
+        True,
+        True,
+        False,
+    ]
+    result = policy.negative_examples("2.1.1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_policy_negative_examples_1(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth = [
+        False,
+        False,
+        True,
+        True,
+        True,
+        True,
+        True,
+        True,
+        # Mutli-label Test cases
+        False,
+        True,
+        True,
+    ]
+    result = policy.negative_examples("1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_policy_negative_examples_2(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth = [
+        True,
+        True,
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        # Mutli-label Test cases
+        True,
+        True,
+        False,
+    ]
+    result = policy.negative_examples("2")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_policy_negative_examples_3(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth = [
+        False,
+        False,
+        False,
+        True,
+        False,
+        False,
+        True,
+        True,
+        # Mutli-label Test cases
+        False,
+        False,
+        True,
+    ]
+    result = policy.negative_examples("2.1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_get_binary_examples_1d_1(digraph, features_1d, labels):
```

Comment:
Perhaps here we might need to diverge the code more. What do you think of applying a logic operation, e.g., `or` or `and`? For example, we could have
```
            [["1", "1.1"], ["1", "1.2"]], # True or True
            [["1", "1.1"], ["2", "2.1"]], # True or False
            [["2.1", "2.1.1"], ["2.2", "2.2.2"]], # False or False
```